content/en/blog/_posts/non-root-containers-and-devices.md

```diff
@@ -0,0 +1,196 @@
+---
+layout: blog
+title: 'Non-root Containers And Devices'
+date: 2021-09-YY
+slug: non-root-containers-and-devices
+---
+
+**Author:** Mikko Ylinen (Intel)
+
+The user/group ID related security settings in Pod's `securityContext` trigger a problem when users want to
+deploy containers that use accelerator devices (via [Kubernetes Device Plugins](docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/)) on Linux. In this blog
+post we talk about the problem and describe the work that was made so far to address it. It's not meant to be a long story about getting the ["k/k issue"](https://github.com/kubernetes/kubernetes/issues/92211) fixed but to raise the awaress of it and important device use-cases too. This is needed as we work on new related features such as support for user namespaces.
```

Comment:
```suggestion
post we talk about the problem and describe the work done so far to address it. It's not meant to be a long story about getting the [k/k issue](https://github.com/kubernetes/kubernetes/issues/92211) fixed. 

Instead,  this post aims to raise awareness of the issue and to highlight important device use-cases too. This is needed as we work on new related features such as support for user namespaces.
```

Reply:
I would replace “we” as there is only one listed author. First person is “I” for a blog article with one author.